src/components/on-this-page/on-this-page.js

```diff
@@ -75,15 +75,15 @@ export default class OnThisPage extends React.PureComponent {
           className={classnames('', {
             none: isChild && !active,
             'block pl12': isChild && active,
-            my6: isChild
+            my3: isChild
```

Comment:
💡 I adjusted the margins to condense them slightly. This better matches the vertical rhythm of NavigationAccordion.